STYLE.md

```diff
@@ -0,0 +1,12 @@
+# Go coding style
+
+* The xDS Relay source code is formatted using
+  [golangci-lint](https://github.com/golangci/golangci-lint). Thus all white spaces, etc.
+  issues are taken care of automatically. Azure pipeline tests will automatically check the code
+  format and fail.
+* More detailed descriptions of each linter can be found
+  [here](https://github.com/golangci/golangci-lint/tree/v1.23.6#enabled-by-default-linters).
```

Comment:
Oops, see this here. We should make our own configuration file and be explicit about what linters we want to run.

Reply:
Yeah we'll definitely want a custom `.golangci.yml` file. This will be added alongside the Go scaffolding and I can update this style guide to point to it once its in place. I think it makes more sense to be done in a future PR, lmk.